package.json

```diff
@@ -35,6 +37,7 @@
     "@typescript-eslint/parser": "^4.29.3",
     "@visx/responsive": "1.1.0",
     "animate.css": "4.1.1",
+    "apollo-client": "2.6.10",
```

Comment:
Is apollo-client needed, when @apollo/client is already used?

Reply:
Good point. There may also be a few other un-needed packages, I'll double check and uninstall

Reply:
@ShynRou it was needed - I think it's used by @apollo/client. I removed it and the code wouldn't compile. Same with the graphql package

Reply:
Pushed it. Will merge once cloudflare is done.